modules/ROOT/pages/salesforce-to-twilio.adoc

```diff
@@ -0,0 +1,454 @@
+
+A Mule application consists of one or more _flows_. According to the MuleSoft glossary, a flow is "a simple, flexible mechanism that enables orchestration of services using the sophisticated message flow capabilities of Mule runtime engine. Also referred to as service orchestration." In Flow Designer, what this means is a sequence of _cards_ that represent core components in Mule Runtime or the operations of connectors. (Cards can represent a couple of other things, too; however, for this tutorial, you need know only about those two.)
+
+Data moves from the beginning of a flow to the end of the flow. Along the way, the cards can update systems, query systems, send messages, delete database rows, or do any of the things that core components and connectors can do.
+
+There is a good number of core components and there are many connectors. For this tutorial, you'll use these:
+
+* Salesforce connector
+* Set Variable component
+* Choice router component
+* Database connector
+* Set Payload component
+* Transform component
+* Twilio connector
+
+For this project, you'll create only a single flow.
+
+Follow these steps to create the project.
+
+. Log into Anypoint Platform.
+. Open Design Center and create a new Mule application.
+.. Click *Create* next to the search field.
+.. In the *Create a Mule Application* section of the overlay, click *Create New Application*.
+.. Name your project `Order-to-cash automation`.
+. In the *Let’s Get Started* wizard, click the link `Go straight to canvas`, which is in the bottom-left corner.
++
+*Result:* Flow Designer appears, an empty trigger card displayed in the middle of the canvas.
++
+image::canvas-empty-trigger.png[]
+
+== Configure the trigger for the flow
+
+A trigger is an event that starts a flow in a Mule application. In your project, the trigger is the change in status of an opportunity in Salesforce Sales Cloud to `Closed Won`. You can configure the trigger card to represent a Salesforce connector that is subscribed to the topic `ClosedOpps`, which you created above in "Capture opportunities that are won".
+
+. Click the Trigger card.
++
+*Result:* A list of the available connectors is displayed in the card.
+. In the *Search* field, type “Salesforce”.
++
+*Result:* The Salesforce connector appears in the list.
+. Select the Salesforce connector.
++
+*Result:* A list of operations that you can use to trigger Mule applications with the Salesforce connector is displayed.
+. Select the operation *Subscribe Topic*.
++
+*Result:* The card is opened. In its top-left corner, it is identified as a Salesforce connector. The name of the operation -- Subscribe Topic -- is also displayed. At the top of the card is the message “You need to set up the Salesforce configuration. Click here to set it up.”
+. Specify how you want Flow Designer to authenticate to Salesforce.
+.. Click the linked text that is in the message.
+.. In the *Connection Type* field, select *Username Password (Deprecated)*.
```

Comment:
@rddheath please remove the line that says *Username Password (Deprecated)* and replace it with
*Basic Username Password*